shell/server/admin-server.js

```diff
@@ -21,6 +21,7 @@ import Heapdump from "heapdump";
 import { SANDSTORM_LOGDIR } from "/imports/server/constants.js";
 import { clearAdminToken, checkAuth, tokenIsValid, tokenIsSetupSession } from "/imports/server/auth.js";
 import { send as sendEmail } from "/imports/server/email.js";
+import { fillUndefinedForChangedDoc } from "/imports/server/observeHelpers.js";
```

Comment:
I'd prefer that the filename here be lower-case-with-hyphens, like the rest of the shell.